lib/checks/aria/braille-label-equivalent.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "braille-label-equivalent",
+  "evaluate": "braille-label-equivalent-evaluate",
+  "metadata": {
+    "impact": "serious",
+    "messages": {
+      "pass": "aria-braillelabel is not used on an element with no accessible text",
```

Just simplifying the double negative.

```suggestion
      "pass": "aria-braillelabel is used on an element with accessible text",
```